_docs/SUMMARY.md

```diff
@@ -11,6 +11,8 @@
         * [Default release](roles/app/default-release.md)
         * [Error pages](roles/app/error-pages.md)
         * [GoCD agent](roles/app/gocd-agent.md)
+        * [SAML](roles/app/saml.md)
```

Comment:
Split out as we discussed. 
Wasn't sure if the debug or error pages role should be moved to moodle only though :thinking: